pandas/tests/frame/test_apply.py

```diff
@@ -1357,3 +1357,14 @@ def test_frequency_is_original(self, num_cols):
         df = DataFrame(1, index=index, columns=range(num_cols))
         df.apply(lambda x: x)
         assert index.freq == original.freq
+
```

Comment:
put with the groupby apply test

needs all of the cases tested that were originally  listed

pls search for open groupby apply issues that are similar

Reply:
By needing all the cases do you mean using separate asserts instead of a single one on the whole result?  I searched for other issues and haven't been able to find any others that this might close.